<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgregarP</class>
 <widget class="QDialog" name="AgregarP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar Producto</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/new/prefix1/Logo.png</normaloff>:/new/prefix1/Logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit">
     <property name="placeholderText">
      <string>Producto</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Precio Compra:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Stock:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Producto:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox">
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Precio Venta:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>doubleSpinBox_2</tabstop>
 </tabstops>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AgregarP</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AgregarP</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
